### ISSUE TITLE
Reload mesh work only once

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#23888]
**Godot version:**
![image](https://user-images.githubusercontent.com/13235119/48857931-06de6580-edb2-11e8-9f68-1f02c670bb53.png)

**OS/device including version:**
win7-64 
![image](https://user-images.githubusercontent.com/13235119/48857969-22497080-edb2-11e8-98fc-682bc8c8a912.png)

**Issue description:**
Godot refresh an imported 3d .escn file only once.
If I export again it godot show me the old one.
To see the correct one I have to close the current and open it again.

**Steps to reproduce:**
Export a simple cube from Blender using "Godot Engine .escn" plugin.
Open it from godot as "New Inherited"
Modify the cube in Blender and export it again.
In godot it will be redraw correctly.
Again go in Blender and change again the previous cube.
Now in Godot show the old modify.
To see the last one I have to close the current and open it again (or open a duplicate - that refresh also the old one)

**Minimal reproduction project:**
NA

**Comment by Calinou** [comment]
Duplicate of #38853.
